### ISSUE TITLE
Tuning the classifier with new dataset

### ISSUE THREAD

**Issue by kameshtj** [facebookresearch/fastText#847]
I have trained a fasttext classifier model with a training set . Now I have new datas for training. So I need to tune the classifier with new dataset.
How can I initialize fasttext classifier with old model ?

**Comment by Celebio** [comment]
Hi @kameshtj ,
We are working on this feature but unfortunately we can't commit to the date of shipping.

For the moment, you can use `-pretrainedVectors` option. This will initialize the word embeddings with the one you provide, but the classifier will be initialized empty (actually with small random numbers). This means it will forget the previous classification task, and learn classification only on the new data.
Actually it won't totally forget the previous classification, the previously trained word vectors will contain some sort of information on it : for example, word that are discriminant for a specific label will be closer to each other. So it may help.

I assume what you exactly need is to fine tune the classifier as well, in which case you have to wait for the feature to be shipped, in the meanwhile you could try this workaround. 

Best regards,
Onur

**Comment by cultivater** [comment]
> Hi @kameshtj , We are working on this feature but unfortunately we can't commit to the date of shipping.
> 
> For the moment, you can use `-pretrainedVectors` option. This will initialize the word embeddings with the one you provide, but the classifier will be initialized empty (actually with small random numbers). This means it will forget the previous classification task, and learn classification only on the new data. Actually it won't totally forget the previous classification, the previously trained word vectors will contain some sort of information on it : for example, word that are discriminant for a specific label will be closer to each other. So it may help.
> 
> I assume what you exactly need is to fine tune the classifier as well, in which case you have to wait for the feature to be shipped, in the meanwhile you could try this workaround.
> 
> Best regards, Onur

Hi @Celebio , is this feature supported now?

**Comment by sinaahmadi** [comment]
I'm also curious to know if this feature is also added. Otherwise, any workaround? 🧐
